test/token/ERC721/ERC721Pausable.test.js

```diff
@@ -6,10 +6,7 @@ const { shouldBehaveLikePublicRole } = require('../../behaviors/access/roles/Pub
 const ERC721PausableMock = artifacts.require('ERC721PausableMock.sol');
 
 contract('ERC721Pausable', function ([
-  _,
```

Comment:
As described in #1718, we shouldn't remove this one so that the first account isn't the `creator` account.

Reply:
got it - added it back in